README.md

```diff
@@ -27,7 +27,7 @@ After you're finished please remove all the comments and instructions!
   <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
```

Comment:
- [OPTIONAL] Your README is well documented, I just noticed that the logo link is broken, 
   so I suggest fixing it 👍